x/mint/types/genesis.go

```diff
@@ -42,5 +47,14 @@ func ValidateGenesis(data GenesisState) error {
 		return ErrInvalidEcosystemTokensMinted
 	}
 
+	if data.MonthsUnlocked.IsNegative() {
+		return ErrInvalidMonthsUnlocked
+	}
+
+	thirtySix := math.NewInt(36)
+	if data.MonthsUnlocked.GT(thirtySix) {
```

Comment:
MonthsUnlocked is defined as an int. Validation could also check < 0 values

Reply:
literally two lines above it checks `IsNegative`